x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/inline_tags_list.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiBadgeGroup, EuiBadge, EuiToolTip } from '@elastic/eui';
+
+import { Query } from '../../types';
+
+interface Props {
+  tags: Query['tags'];
+}
+export const InlineTagsList: React.FC<Props> = ({ tags }) => {
+  if (!tags?.length) return null;
```

Comment:
Oh that's interesting. Looking above it doesn't appear that `tags` is optional, but I guess it is, because that's how `tags` is defined on the `Query` type.

Reply:
I can definitely update `tags:` on line 14 to `tags?:` to make that more explicit if that helps!

Reply:
4751eb7